hms-app/src/components/HeaderMenu.tsx

```diff
@@ -65,8 +66,19 @@ export default function HeaderMenu({
   today.setHours(0, 0, 0, 0)
   const location = useLocation()
   const { instance, accounts } = useMsal()
-  const user = accounts.length > 0 ? accounts[0] : null
   const [editModalOpened, setEditModalOpened] = useState(false)
+  const [user, setUser] = useState({
+    id: 'string',
+    lastName: 'string',
+    firstName: 'string',
+    emailAddress: 'string',
+    roles: [],
+    skills: [],
+    imageUrl: 'string',
+    creationDate: new Date(),
+  } as User)
+  const [isUserError, setIsUserError] = useState(false)
+  const [isUserLoading, setIsUserLoading] = useState(true)
```

Comment:
both states are never used

Reply:
do we need it?

Reply:
maybe we should wrap the return statement with this error and loading checks? so that if something goes wrong we can display a "whoopsie" message. 
take a look at UserDetails line 111